Controller/SitemapController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
```

Comment:
it's 2015 now :) (and as this file was added in 2015, it should officialy say "(c) 2015 Symfony CMF")